src/Compilers/Lua/Portable/LuaSyntaxOptions.cs

```diff
@@ -192,7 +195,8 @@ public class LuaSyntaxOptions : IEquatable<LuaSyntaxOptions?>
             decimalIntegerFormat: IntegerFormats.NotSupported,
             hexIntegerFormat: IntegerFormats.NotSupported,
             acceptTypedLua: true,
-            acceptFloorDivision: false);
+            acceptFloorDivision: false,
+            acceptLuaJITNumberSuffixes: false);
```

Comment:
The All preset should accept anything as long as it doesn't conflict with what it had before